README.md

```diff
@@ -47,7 +47,6 @@ have to leave Neovim!
 
 - System-wide
   - curl
-  - jq +1.6 (to format JSON output so it can be human-readable)
```

Comment:
Maybe keep it as an optional dependency?

Also I think tidy should be added as an optional dependency too as AFAIK it is an external tool right?

Reply:
@NTBBloodbath

`jq` and `tidy` can be replaced by other executable or even native lua functions, that's why I didn't add them as dependencies.

Reply:
I don't think they should be added as optional dependencies as people would just install them thinking they are required.

Reply:
Ah, makes sense now. Yeah let's leave it as this then, thanks for explaining